ast/edit/option_editor.go

```diff
@@ -152,15 +156,16 @@ type optionEditor struct {
 
 func (v *optionEditor) Visit(node ast.Node) ast.Visitor {
 	if os, ok := node.(*ast.OptionStatement); ok {
-		if os.Assignment.ID.Name == v.identifier {
+		// Cannot edit namespace qualified options
```

Comment:
Why not? We are editing the AST it seems practical to be able to edit qualified options.

Reply:
Wasn't sure if it was allowed so decided not to change it. I can modify this now or merge and modify as a separate PR.